SCClassLibrary/JITLib/ProxySpace/ProxyInterfaces.sc

```diff
@@ -231,7 +232,17 @@ SynthControl : AbstractPlayControl {
 		server = target.server;
 		group = target.asTarget;
 		nodeID = server.nextNodeID;
-		bundle.addCancel({ [9, this.asDefName, nodeID, addAction, group.nodeID] ++ extraArgs.value });
+
+		bundle.addCancel({
+			if (blockSize.notNil or: { upsample.notNil }) {
+				[66, //"s_newEx"
+					this.asDefName, nodeID, addAction, group.nodeID,
+					blockSize ? 0, upsample ? 0.0]
```

Comment:
you can leave out the `? 0` in all the code – nil is read as zero in the server.

Reply:
I know, I left this in for clarity and readability.